src/FluentCMS.Web.UI/static/scss/components/_dropdown.scss

```diff
@@ -26,11 +26,12 @@ $prefix-dropdown: $prefix + 'dropdown';
     }
 
     &-item-link {
-        @apply block px-4 py-2; 
+        @apply block px-4 py-2;
     }
 
     &-menu {
-        @apply absolute z-10 bg-white divide-y divide-gray-100 rounded-lg shadow w-44;
+        @apply absolute z-10 bg-white divide-y divide-gray-100 rounded-lg shadow;
+        min-width: 11rem;
```

Comment:
Couldn’t we use Tailwind classes to do that?

Reply:
no, I tried `min-w-44` which should be equivalent to this code, bud this class doesn't exist in tailwind